src/vm/profilinghelper.cpp

```diff
@@ -1049,14 +1049,6 @@ HRESULT ProfilingAPIUtility::LoadProfiler(
 
         _ASSERTE(profilerCompatibilityFlag == kEnableV2Profiler);
 
-        // To prevent V2 profilers from AV, once a V2 profiler is already loaded by a V2 rutnime in the process, 
-        // V4 runtime will not try to load the V2 profiler again.
-        if (IsV2RuntimeLoaded())
-        {
-            LogProfInfo(IDS_PROF_V2PROFILER_ALREADY_LOADED, wszClsid);
```

Comment:
I removed the display, but not the actual resource. What is the process here to remove the resource string? Just edit the resource files manually?

Reply:
cc @noahfalk

Reply:
> Just edit the resource files manually?

Yes.